Data Structures/Arrays/Arrays - DS/solution.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const fs = require('fs');
+
+process.stdin.resume();
+process.stdin.setEncoding('utf-8');
+
+let inputString = '';
+let currentLine = 0;
+
+process.stdin.on('data', inputStdin => {
+    inputString += inputStdin;
+});
+
+process.stdin.on('end', function() {
+    inputString = inputString.replace(/\s*$/, '')
+        .split('\n')
+        .map(str => str.replace(/\s*$/, ''));
+
+    main();
+});
+
+function readLine() {
+    return inputString[currentLine++];
+}
+
+
+// Helper function: just swap two places in the array
+function swap(array, a, b) {
+    let tmp = array[a];
+    array[a] = array[b];
+    array[b] = tmp;
+}
+// In-place reverse.
+function reverseArray(a) {
+    for (let i = 0; i < a.length / 2; i += 1) {
+        swap(a, i, a.length - 1 - i);
+    }
+    return a;
+}
+
+function main() {
+    const ws = fs.createWriteStream(process.env.OUTPUT_PATH);
+
```

Comment:
Good solution! Just remove the extra lines in between so we have a better formatting.

Reply:
If I understood you correctly, done.